pkg/client/cli/cmds_intercept.go

```diff
@@ -151,6 +159,13 @@ func interceptCommand(ctx context.Context) *cobra.Command {
 
 	flags.StringVarP(&args.namespace, "namespace", "n", "", "If present, the namespace scope for this CLI request")
 
+	flags.BoolVarP(&args.cmdLineIngress, "cmd-ingress", "i", false, "if set, uses cmd flags instead of opening a dialogue for ingress settings."+
+		"Uses --ingressHost, --ingressPort, --ingressTLS, and --ingressL5")
+	flags.StringVar(&args.ingressHost, "ingressHost", "", "if cmd-ingress is set, this value will be used as the ingress hostname")
```

Comment:
I suggest you use all lower-case and dash separated words. All current command flags follow those rules.